### DISCUSSION TITLE
NextJS 13 Redirect from Server Side Fetch Function

### DISCUSSION THREAD

**Discussion by muellerluke** [vercel/next.js#42833] [Help]
Is there a way to redirect from a server side fetch function like there was in NextJS 12? Previously, you could just return the redirect object instead of props from a getServerSideProps function, but how would you go about this now? Is there a way to do this now?

**Comment by mrksph** [comment]
Any new about this?

In my API I have an endpoint that I'd like to return a 30X redirect with a Location header.

When I fetch from my server-side component I don't know how to follow the redirect. As far as I understand I can only redirect manually.

Can someone shed some light on this?

Thanks
